changelog/next/features/5177--from-http-client.md

```diff
@@ -0,0 +1 @@
+We implemented the HTTP client for the `from_http` operator.
```

Comment:
What does this mean for `load_http` and `from "http://…"`?

Reply:
Nothing?